packages/realm-web/src/services/MongoDBService.ts

```diff
@@ -229,6 +229,11 @@ class MongoDBCollection<T extends Document>
             upsert: options.upsert,
         });
     }
+
+    /** @inheritdoc */
+    watch(): AsyncGenerator<any> {
```

Comment:
Do you want to throw the `async*` on here now and put in a defaulted arg? I don't think it makes a huge difference either way, it just seems worth making the signature correct now, even if you aren't implementing yet.

Reply:
Adding the options argument, but adding the `async*` yields a eslint warinng (so I'll leave that out for now):
> This generator function does not have 'yield'. `eslint(require-yield)`